plugins/woocommerce-admin/client/two-column-tasks/style.scss

```diff
@@ -241,6 +240,11 @@
 		flex: 1;
 		overflow: hidden;
 		width: 100%;
+
+		img {
+			max-height: 150px;
+			width: auto;
+		}
```

Comment:
Not sure if this works well for all the tasks in the two column layout, I screwed around with that for a while to, maybe we still want to add a max-width?
<img width="611" alt="Screen Shot 2022-06-01 at 4 15 23 PM" src="https://user-images.githubusercontent.com/2240960/171484373-1bb0eb0c-0b5e-4530-807a-854572f214fb.png">

Reply:
Add max width and max height.  Seems to be working well on my end, can you double check on yours?

Reply:
Looks good on my end now to :)